### PR TITLE
chore(website): Monthly Product updates 3

### PR DESCRIPTION
**Pull request by yevgenypats**
Quite nice to look back only 4 weeks and realize how much is done 🚀  Kudos to everyone !

### CODE REVIEW

File: website/pages/blog/product-updates-3.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+### New Sources
+
+We've added 7 new source plugins to CQ to cover the long tail of infrastructure apps!
+
+* [Slack](https://www.cloudquery.io/docs/plugins/sources/slack/overview)
+* [PagerDuty](https://www.cloudquery.io/docs/plugins/sources/pagerduty/overview)
+* [Alibaba Cloud](https://www.cloudquery.io/docs/plugins/sources/alicloud/overview)
+* [Oracle](https://www.cloudquery.io/docs/plugins/sources/oracle/overview)
+* [GitLab](https://www.cloudquery.io/docs/plugins/sources/gitlab/overview)
+* [HackNews](https://www.cloudquery.io/docs/plugins/sources/hackernews/overview)
```
Comment (erezrokah) [inline website/pages/blog/product-updates-3.md:46]:
```suggestion
* [Hacker News](https://www.cloudquery.io/docs/plugins/sources/hackernews/overview)
```

File: website/pages/blog/product-updates-3.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+* [File](https://www.cloudquery.io/docs/plugins/destinations/file/overview)
+* [Azure Blob Storage](https://www.cloudquery.io/docs/plugins/destinations/azblob/overview)
+
+## Sources
+
+### New Sources
+
+We've added 7 new source plugins to CQ to cover the long tail of infrastructure apps!
+
+* [Slack](https://www.cloudquery.io/docs/plugins/sources/slack/overview)
```
Comment (erezrokah) [inline website/pages/blog/product-updates-3.md:41]:
```suggestion
* [Slack](/docs/plugins/sources/slack/overview)
```

File: website/pages/blog/product-updates-3.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+Hey everyone! Our monthly round-up of CloudQuery product and roadmap updates is here! This month was packed!
+
+## SDK Updates
+
+- We now support [incremental syncing](https://www.cloudquery.io/docs/core-concepts/syncs#incremental-table-syncs) in alpha! This allow you to sync only new data from your sources. This is a huge performance improvement for large tables.
```
Comment (erezrokah) [inline website/pages/blog/product-updates-3.md:18]:
```suggestion
- We now support [incremental syncing](/docs/core-concepts/syncs#incremental-table-syncs) in alpha! This allow you to sync only new data from your sources. This is a huge performance improvement for large tables.
```

File: website/pages/blog/product-updates-3.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+## Destinations
+
+### New Destinations
+
+We've added 7 new destinations to CQ!!
+
+* [MongoDB](https://www.cloudquery.io/docs/plugins/destinations/mongodb/overview)
+* [Neo4j](https://www.cloudquery.io/docs/plugins/destinations/neo4j/overview)
+* [Kafka](https://www.cloudquery.io/docs/plugins/destinations/kafka/overview)
```
Comment (erezrokah) [inline website/pages/blog/product-updates-3.md:28]:
```suggestion
* [Kafka](/docs/plugins/destinations/kafka/overview)
```

File: website/pages/blog/product-updates-3.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+* [MSSQL](https://www.cloudquery.io/docs/plugins/destinations/mssql/overview)
+* [GCS](https://www.cloudquery.io/docs/plugins/destinations/gcs/overview)
```
Comment (erezrokah) [inline website/pages/blog/product-updates-3.md:30]:
```suggestion
* [GCS](/docs/plugins/destinations/gcs/overview)
```

File: website/pages/blog/product-updates-3.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+* [S3](https://www.cloudquery.io/docs/plugins/destinations/s3/overview)
```
Comment (erezrokah) [inline website/pages/blog/product-updates-3.md:31]:
```suggestion
* [S3](/docs/plugins/destinations/s3/overview)
```

File: website/pages/blog/product-updates-3.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+* [File](https://www.cloudquery.io/docs/plugins/destinations/file/overview)
```
Comment (erezrokah) [inline website/pages/blog/product-updates-3.md:32]:
```suggestion
* [File](/docs/plugins/destinations/file/overview)
```

File: website/pages/blog/product-updates-3.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+* [Azure Blob Storage](https://www.cloudquery.io/docs/plugins/destinations/azblob/overview)
```
Comment (erezrokah) [inline website/pages/blog/product-updates-3.md:33]:
```suggestion
* [Azure Blob Storage](/docs/plugins/destinations/azblob/overview)
```

File: website/pages/blog/product-updates-3.md [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+
+## Sources
+
+### New Sources
+
+We've added 7 new source plugins to CQ to cover the long tail of infrastructure apps!
```
Comment (erezrokah) [inline website/pages/blog/product-updates-3.md:39]:
```suggestion
We've added 8 new source plugins to CQ to cover the long tail of infrastructure apps!
```

File: website/pages/blog/product-updates-3.md [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+
+* [Slack](https://www.cloudquery.io/docs/plugins/sources/slack/overview)
+* [PagerDuty](https://www.cloudquery.io/docs/plugins/sources/pagerduty/overview)
```
Comment (erezrokah) [inline website/pages/blog/product-updates-3.md:42]:
```suggestion
* [PagerDuty](/docs/plugins/sources/pagerduty/overview)
```

File: website/pages/blog/product-updates-3.md [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+* [Alibaba Cloud](https://www.cloudquery.io/docs/plugins/sources/alicloud/overview)
```
Comment (erezrokah) [inline website/pages/blog/product-updates-3.md:43]:
```suggestion
* [Alibaba Cloud](/docs/plugins/sources/alicloud/overview)
```

File: website/pages/blog/product-updates-3.md [hunk 12]
```diff
@@ -0,0 +1,64 @@
...
+* [Oracle](https://www.cloudquery.io/docs/plugins/sources/oracle/overview)
```
Comment (erezrokah) [inline website/pages/blog/product-updates-3.md:44]:
```suggestion
* [Oracle](/docs/plugins/sources/oracle/overview)
```

File: website/pages/blog/product-updates-3.md [hunk 13]
```diff
@@ -0,0 +1,64 @@
...
+* [GitLab](https://www.cloudquery.io/docs/plugins/sources/gitlab/overview)
```
Comment (erezrokah) [inline website/pages/blog/product-updates-3.md:45]:
```suggestion
* [GitLab](/docs/plugins/sources/gitlab/overview)
```

File: website/pages/blog/product-updates-3.md [hunk 14]
```diff
@@ -0,0 +1,64 @@
...
+* [HackNews](https://www.cloudquery.io/docs/plugins/sources/hackernews/overview)
+* [Shopify](https://www.cloudquery.io/docs/plugins/sources/shopify/overview)
```
Comment (erezrokah) [inline website/pages/blog/product-updates-3.md:47]:
```suggestion
* [Shopify](/docs/plugins/sources/shopify/overview)
```

File: website/pages/blog/product-updates-3.md [hunk 15]
```diff
@@ -0,0 +1,64 @@
...
+* [Stripe](https://www.cloudquery.io/docs/plugins/sources/stripe/overview)
```
Comment (erezrokah) [inline website/pages/blog/product-updates-3.md:48]:
```suggestion
* [Stripe](/docs/plugins/sources/stripe/overview)
```

File: website/pages/blog/product-updates-3.md [hunk 16]
```diff
@@ -0,0 +1,64 @@
...
+
+Numerous updates to [AWS](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/aws/CHANGELOG.md), [GCP](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/gcp/CHANGELOG.md) and [Azure](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/azure/CHANGELOG.md).
+
+## Other notable blogs and use-cases
+
+### Engineering
+
+- [Reducing Alert Fatigue with the PagerDuty Plugin](https://www.cloudquery.io/blog/announcing-pagerduty-plugin).
```
Comment (erezrokah) [inline website/pages/blog/product-updates-3.md:56]:
```suggestion
- [Reducing Alert Fatigue with the PagerDuty Plugin](/blog/announcing-pagerduty-plugin).
```

File: website/pages/blog/product-updates-3.md [hunk 17]
```diff
@@ -0,0 +1,64 @@
...
+
+### Security
+
+- [How to use CloudQuery for Attack Surface Management and Graph Visualization](https://www.cloudquery.io/how-to-guides/attack-surface-management-with-graph)
```
Comment (erezrokah) [inline website/pages/blog/product-updates-3.md:60]:
```suggestion
- [How to use CloudQuery for Attack Surface Management and Graph Visualization](/how-to-guides/attack-surface-management-with-graph)
```

File: website/pages/blog/product-updates-3.md [hunk 18]
```diff
@@ -0,0 +1,64 @@
...
+
+### FinOps
+
+- [https://www.cloudquery.io/blog/azure-cost-optimization-with-cloudquery](https://www.cloudquery.io/blog/azure-cost-optimization-with-cloudquery)
```
Comment (erezrokah) [inline website/pages/blog/product-updates-3.md:64]:
```suggestion
- [Correlating Costs to Resources for Azure Cost Optimization with CloudQuery](/blog/azure-cost-optimization-with-cloudquery)
```

File: website/pages/blog/product-updates-3.md [hunk 19]
```diff
@@ -0,0 +1,64 @@
...
+* [Slack](https://www.cloudquery.io/docs/plugins/sources/slack/overview)
+* [PagerDuty](https://www.cloudquery.io/docs/plugins/sources/pagerduty/overview)
+* [Alibaba Cloud](https://www.cloudquery.io/docs/plugins/sources/alicloud/overview)
+* [Oracle](https://www.cloudquery.io/docs/plugins/sources/oracle/overview)
+* [GitLab](https://www.cloudquery.io/docs/plugins/sources/gitlab/overview)
+* [HackNews](https://www.cloudquery.io/docs/plugins/sources/hackernews/overview)
+* [Shopify](https://www.cloudquery.io/docs/plugins/sources/shopify/overview)
+* [Stripe](https://www.cloudquery.io/docs/plugins/sources/stripe/overview)
+
+Numerous updates to [AWS](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/aws/CHANGELOG.md), [GCP](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/gcp/CHANGELOG.md) and [Azure](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/azure/CHANGELOG.md).
```
Comment (erezrokah) [inline website/pages/blog/product-updates-3.md:50]:
```suggestion
Numerous updates to [AWS](https://github.com/cloudquery/cloudquery/blob/40ff6b263b4ad0e96b00495731ed8f4247f87b52/plugins/source/aws/CHANGELOG.md), [GCP](https://github.com/cloudquery/cloudquery/blob/40ff6b263b4ad0e96b00495731ed8f4247f87b52/plugins/source/gcp/CHANGELOG.md) and [Azure](https://github.com/cloudquery/cloudquery/blob/40ff6b263b4ad0e96b00495731ed8f4247f87b52/plugins/source/azure/CHANGELOG.md).
```

We should pin these files to a specific commit so their content reflects the time the blog is published (you can do this by clicking `y` on your keyboard when you have a file open)

File: website/pages/blog/product-updates-3.md (hunk 3 above)
Comment (erezrokah) [inline website/pages/blog/product-updates-3.md:18]:
```suggestion
- We now support [incremental syncing](/docs/core-concepts/syncs#incremental-table-syncs) in alpha! This allow you to sync only new data from your sources. This is a huge performance improvement for large tables.
```

**Review by erezrokah — APPROVED** [review]
Looks great, comments are not blocking, only best practices

File: website/pages/blog/product-updates-3.md [hunk 20]
```diff
@@ -0,0 +1,64 @@
...
+
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+Hey everyone! Our monthly round-up of CloudQuery product and roadmap updates is here! This month was packed!
+
+## SDK Updates
+
+- We now support [incremental syncing](/docs/core-concepts/syncs#incremental-table-syncs) in alpha! This allow you to sync only new data from your sources. This is a huge performance improvement for large tables.
```
Comment (hermanschaaf) [inline website/pages/blog/product-updates-3.md:18]:
```suggestion
- We now support [incremental syncing](/docs/core-concepts/syncs#incremental-table-syncs) in alpha! This allows you to sync only new data from your sources. This is a huge performance improvement for large tables.
```

File: website/pages/blog/product-updates-3.md [hunk 21]
```diff
@@ -0,0 +1,64 @@
...
+### New Sources
+
+We've added 7 new source plugins to CQ to cover the long tail of infrastructure apps!
+
+* [Slack](/docs/plugins/sources/slack/overview)
+* [PagerDuty](https://www.cloudquery.io/docs/plugins/sources/pagerduty/overview)
+* [Alibaba Cloud](https://www.cloudquery.io/docs/plugins/sources/alicloud/overview)
+* [Oracle](https://www.cloudquery.io/docs/plugins/sources/oracle/overview)
+* [GitLab](https://www.cloudquery.io/docs/plugins/sources/gitlab/overview)
+* [Hacker News](https://www.cloudquery.io/docs/plugins/sources/hackernews/overview)
```
Comment (hermanschaaf) [inline website/pages/blog/product-updates-3.md:46]:
```suggestion
* [Hacker News](/docs/plugins/sources/hackernews/overview)
```

**Review by hermanschaaf — APPROVED** [review]
🚀

**Pull request merged**